lib/mapper.js

```diff
@@ -189,14 +205,18 @@ function properties(track){
 
 function traits(identify){
```

Comment:
separate, esp. considering everything looks :ok: in their interface with the existing setup